assets/general/bitcoin/info.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Bitcoin",
+  "website": "https://bitcoin.org",
+  "description": "Bitcoin is a cryptocurrency and worldwide payment system. It is the first decentralized digital currency, as the system works without a central bank or single administrator.",
+  "explorer": "https://blockchain.info",
+  "research": "https://research.binance.com/en/projects/bitcoin",
+  "symbol": "BTC",
+  "type": "coin",
+  "decimals": 8,
+  "minBalance": 0.00001,
+  "cryptoTransferDecimals": 8,
+  "minTransferAmount": 0.00000546,
+  "fixedFee": 0.00003153,
+  "status": "active",
+  "links": [
+      {
```

Comment:
Non consistent tabulation. Expected tab with 2 spaces instead of 4 spaces. Check other files as well.